v3/src/lib/handle-cfm-event.ts

```diff
@@ -66,10 +94,22 @@ export function handleCFMEvent(cfmClient: CloudFileManagerClient, event: CloudFi
       }
       break
     }
-    // case "sharedFile":
-    //   break
-    // case "unsharedFile":
-    //   break
+    case "sharedFile":
+      // Trigger a save so the sharing info can be updated in the file
+      // If the file is already shared, and the user updates the shared document this event
+      // will happen again. Currently it isn't necessary to update the sharing info, but
+      // perhaps in the future the sharing info will include properties that change each
+      // time, such as a timestamp for when the document was shared.
+      // Due to the design of the CFM event system we need to do this in the next time slice
+      await new Promise(resolve => setTimeout(resolve, 0))
+      cfmClient.dirty(true)
+      break
```

Comment:
```suggestion
```
Since the response to `sharedFile` and `unsharedFile` is the same, seems like they could be handled together.